daml-script/export/src/main/scala/com/daml/script/export/Encode.scala

```diff
@@ -142,9 +142,14 @@ private[export] object Encode {
       cidMap: Map[ContractId, String],
       v: Value.Sum,
   ): Doc = {
+    def isTupleRecord(recordId: Identifier): Boolean = {
```

Comment:
given that these are all in a stable package, maybe worth checking the package id as well?

Reply:
Done.